ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14-periodics.yaml

```diff
@@ -27948,18 +27948,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-openshift-logging-5.8-gcp-ipi-sdn-fips-f3
-  reporter_config:
-    slack:
-      channel: '#aos-qe-logging'
```

Comment:
Slack notification removed. Is it expected ?

Reply:
Oh, no, I'll add it back. Thanks for capturing it.